### PR TITLE
Blog app - controllers specs

### PR DESCRIPTION
**Pull request by hamid-murambiwa**
- Added RSpec as a gem to your project.
- Created a Request spec file for all my controllers.
- Make sure that for each action I check:
- [ ] If response status was correct.
- [ ] If a correct template was rendered.
- [ ] If the response body includes correct placeholder text.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,80 @@
...
+```
+
+
+
+### Prerequisites
+
+- Ruby
+- Ruby on Rails
+- Postgresql
+
```
Comment (elisha2kyakpo1) [inline README.md:66]: - [ ] Let's include command(s) for testing the app, we should let users know how to test our app in the readme file.

File: spec/requests/posts_spec.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      expect(response).to have_http_status(:ok)
+    end
+
+    it "renders 'index' template" do
+      expect(response).to render_template('show')
+    end
+
+    it "renders 'index' template with template in body" do
+      expect(response.body).to match('Show')
+    end
```
Comment (elisha2kyakpo1) [inline spec/requests/posts_spec.rb]: - [ ] [OPTIONAL] Let's correct this information here, we are testing the show action to render its view template. The text should read "renders show template". This is the same case for line 31.

File: spec/requests/users_spec.rb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    end
+
+    it "renders 'index' template" do
+      expect(response).to render_template('show')
+    end
+
+    it "renders 'index' template with template in body" do
+      expect(response.body).to match('Show Users')
+    end
+  end
```
Comment (elisha2kyakpo1) [inline spec/requests/users_spec.rb]: - [ ] [OPTIONAL] Let's correct this information here, we are testing the show action to render its view template. The text should read "renders show template". This is the same case for line 31.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi @hamid-murambiwa,

Great job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
All the tests are passing 👍

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (rloterh) [reply README.md:66]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Hamid, 

Your project is complete! Great job for making the necessary changes recommended in the previous review. There is nothing else to say other than... it's time to merge it 🍾 :shipit:. Congratulations! 🎉🥂

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**